src/utils/dato.ts

```diff
@@ -1,3 +1,47 @@
+import moment from 'moment';
+import 'moment/min/locales';
+import 'moment/locale/nb';
+moment.locale('nb');
+
+export const STANDARD_DATOFORMAT = 'DD.MM.YYYY';
+export const FØDSELSNUMMER_DATOFORMAT = 'DDMMYY';
```

Comment:
Man kan ikke ta fødselsdato fra fnr. Det er ikke alltid riktig.

Reply:
Det brukes andre veien, for å lage fødselsnummer av dato + personnummer for nye barn man legger til. Slik: `const fødselsnummer = barnDato && personnummer ? formatDateFnr(barnDato) + personnummer : "";`

Reply:
Åja, men kan man stole på det? Henning S har hvertfall gitt beskjed om at det ikke kan brukes.

Reply:
Åja, da stemmer sikkert det. Undersøker.

Reply:
Snakket med Henning S. Det at det ikke alltid stemmer gjelder visst ca 10000 eldre personer som fikk datoen 01.01 da de kom til Norge, men det er visst en safe antagelse for nyfødte.

Reply:
ok 👍